packages/components/src/ui/context/wordpress-component.ts

```diff
@@ -3,19 +3,26 @@
  */
 import type * as React from 'react';
 
-/**
- * Based on https://github.com/reakit/reakit/blob/master/packages/reakit-utils/src/types.ts
- *
- * The `children` prop is being explicitely omitted since it is otherwise implicitly added
- * by `ComponentPropsWithRef`. The context is that components should require the `children`
- * prop explicitely when needed (see https://github.com/WordPress/gutenberg/pull/31817).
- */
```

Comment:
Moved these from JSDoc to normal comments because they're more about implementation details and not useful in IntelliSense.